### ISSUE TITLE
Group create is not working

### ISSUE THREAD

**Issue by spatten** [discourse/discourse_api#148]
I just tried using the `create_group` method against a brand new install of Discourse.

```
client = DiscourseApi::Client.new('http://discourse.dev')
client.api_key = ENV['DISCOURSE_API_KEY']
client.api_username = 'scott'
client.create_group(name: 'newgroup')
```

I get back errors that say something like this:

````
DiscourseApi::Error: ActionController::ParameterMissing at /admin/groups
===================================================

> param is missing or the value is empty: group

app/controllers/admin/groups_controller.rb, line 194
----------------------------------------------------

``` ruby
  189     end
  190
  191     private
  192
  193     def group_params
> 194       params.require(:group).permit(
  195         :name,
  196         :mentionable_level,
  197         :messageable_level,
  198         :visibility_level,
  199         :automatic_membership_email_domains,
````

It looks like the `create_group` method should be passing up `group[name]` instead of `name` as a param.

If I make a PR that fixes this and also adds the other params as optional params, would you be interested in it?

Thanks!

**Comment by coding-horror** [comment]
Sure PRs always welcome! Thanks!!

**Comment by spatten** [comment]
This was fixed when you merged #149. Thanks!
